tests/e2e/helpers/getMysqlAdminPrismaClient.ts

```diff
@@ -0,0 +1,17 @@
+import { PrismaClient } from '@prisma/client'
+import { PrismaClientOptions } from '@prisma/client/runtime'
+
+export type PrismaClientConstructor = new (options: PrismaClientOptions) => PrismaClient
+
+export async function GetMysqlAdminPrismaClient(
+  databaseUrlBase: string,
+  CtxPrismaClient: PrismaClientConstructor
+): Promise<PrismaClient> {
+  return new CtxPrismaClient({
+    datasources: {
+      db: {
+        url: `${databaseUrlBase}/mysql`,
+      },
+    },
+  })
+}
```

Comment:
Let's keep the database admin prisma client connection logic encapsulated in `testTemplate`.